src/SignalRServiceExtension/TriggerBindings/SignalRTriggerBindingProvider.cs

```diff
@@ -62,22 +62,29 @@ internal SignalRTriggerAttribute GetParameterResolvedAttribute(SignalRTriggerAtt
             if (declaredType != null && declaredType.IsSubclassOf(typeof(ServerlessHub)))
             {
                 // Class based model
+                if (!string.IsNullOrEmpty(hubName) ||
```

Comment:
make the properties' setters access level less than public?

Reply:
or move them to another class?

Reply:
Removing Setters for these properties is good.